### ISSUE TITLE
Cannot set custom snooze date when in overlow menu as button is missing

### ISSUE THREAD

**Issue by rugk** [bwinton/SnoozeTabs#487]
## STR

1. Put the button into the overflow menu (i.e. that >> icon).
2. Open it there and open it and select to "Pick a custom date/time".

### What happens

The height is way too large and thus, you cannot click/choose the button to accept the date/time you've entered.

![image](https://user-images.githubusercontent.com/11966684/129632027-bbb2e133-5ab5-4a24-b4f9-d754673e043a.png)
![image2](https://user-images.githubusercontent.com/11966684/129632124-871f6b0d-c18a-4ecc-abbc-40e046fbae92.png)

### What should happen

I should see the buttons and be able to click them.

## Workaround

You can zoom in and suddenly you will see some buttons at the bottom.

Or of course move it into the toolbar or so.

## Potential solution

I know this menu is a hard weird thing and so on. Maybe you can just set a height as a workaround (for now)?

## System

Firefox 90.0.2
Fedora 34, GNOME

**Comment by bwinton** [comment]
Are you still seeing this? I checked in Fx91 on MacOS, and can't reproduce it…

**Comment by rugk** [comment]
Yes I do see this with Firefox 91.0.1 in Fedora 34, GNOME, as before. The screen resolution is full HD _and_ I have a long list of other add-on's/buttons (20 ones) listed in the overflow menu (in case that should affect it). reproducible on two devices. (the other device does have a full HD screen but with HiDPI 125%, so even "less" pixel if you'd say it like this.)

**Comment by bwinton** [comment]
I still can't seem to replicate this. 😢
Could you go to about:addons, click "Debug Add-ons", and then "Inspect" SnoozeTabs, to try and figure out what's going wrong?
<img width="359" alt="Screen Shot 2021-08-29 at 19 00 06" src="https://user-images.githubusercontent.com/159571/131268007-b505290b-205f-46c2-a71f-1a2b32346fd5.png">

You may also need to "Disable Pop-up Autohide" in the meatball menu…
<img width="244" alt="Screen Shot 2021-08-29 at 19 01 25" src="https://user-images.githubusercontent.com/159571/131268059-df7efde4-0f61-4bcb-86c5-68c6bf48e37f.png">

**Comment by rugk** [comment]
Crazy, though I found sth.:
![grafik](https://user-images.githubusercontent.com/11966684/131395051-8fe538ee-5cd4-475a-8cd2-d063f11f4e1d.png)

ooks like some flexbox problem.

`.times` does seem to have a proper height.

Also though, if I remove that `flex-grow: 2`, it also does not work.

This here seems to fix it:
```patch

/* snooze.css | moz-extension://4d40debc-96f8-420b-91c3-418896754013/popup/snooze.css */

.panel > .content {
  /* flex-grow: 2; */
}

/* Element | moz-extension://4d40debc-96f8-420b-91c3-418896754013/popup/snooze.html */

#main {
  height: auto;
}
```

**Comment by rugk** [comment]
Okay, strange diff Firefox gave me, it creates a CSS and no patch, ok.

But here, that `#main` thing yet alone seems to fix it:
```css

/* snooze.css | moz-extension://4d40debc-96f8-420b-91c3-418896754013/popup/snooze.css */

.panel.static.active {
  height: auto;
}
```

It looks/better now/goes into the correct direction:
![grafik](https://user-images.githubusercontent.com/11966684/131396076-ccad39aa-d112-481c-8483-f34c6104ae76.png)

Note that Firefox [popup size system is somewhat complicated](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Popups#Popup_resizing). In any case, once you have a popup as large as it is (the height in this case), it does not shrink it, it just keeps it's height.
That's why I guess the above result looks good. Without really compiling and testing a local copy of the extension I cannot be fully sure though, of course.
